.circleci/config.yml

```diff
@@ -13,13 +13,18 @@ commands:
           command: |
             curl -o Miniconda3-py38_4.8.3-Linux-x86_64.sh https://repo.anaconda.com/miniconda/Miniconda3-py38_4.8.3-Linux-x86_64.sh
             bash ./Miniconda3-py38_4.8.3-Linux-x86_64.sh -b
+      - run:
+          name: Preparing environment - Other Dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y expect fish zsh openjdk-11-jre rsync
```

Comment:
you only need a java runtime to install Hydra.
the rest (fish, zsh, rsync) are used by various Hydra tests you are not running.

Reply:
right, wasnt sure so I included them.